x-pack/plugins/siem/public/common/components/drag_and_drop/drag_drop_context_wrapper.tsx

```diff
@@ -82,7 +83,7 @@ const onDragEndHandler = ({
       browserFields,
       dispatch,
       result,
-      timelineId: ACTIVE_TIMELINE_REDUX_ID,
+      timelineId: getTimelineIdFromColumnDroppableId(result.destination?.droppableId ?? ''),
```

Comment:
I think `reArrangeProviders` needs also the timelineId to be dynamic. I tested locally and the reordering of the columns does not work as expected. I added this line to `reArrangeProviders` and it worked. I am wondering if I am doing something wrong as your cypress test passes. What do you think?

Reply:
My mistake! I had a caching issue. I cleared my cache and it worked as expected. Sorry for that.